src/go/rdctl/cmd/shell.go

```diff
@@ -117,24 +127,59 @@ const restartDirective = "Either run 'rdctl start' or start the Rancher Desktop
 
 func checkLimaIsRunning(commandName string) bool {
 	// Ignore error messages; none are expected here
-	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	cmd := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}")
+	stderr := &cmdWriter{}
+	cmd.Stderr = stderr
+
+	stdout := &cmdWriter{}
+	cmd.Stdout = stdout
+
+	err := cmd.Start()
+	if err == nil {
+		err = cmd.Wait()
+	}
```

Comment:
Why do you need a custom `cmdWriter`, doesn't a simple `bytes.Buffer` do the same job?

```suggestion
    var stdout, stderr bytes.Buffer
	cmd := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}")
	cmd.Stdout = &stdout
	cmd.Stderr = &stderr

	err := cmd.Run()
```